<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title></title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/paper.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/default.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>





<link rel="stylesheet" href="style.css" type="text/css" />



<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html"><font size="7">Joyce R Prado</font></a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="contact.html">
    <span class="fas fa-envelope fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="http://github.com/joycepra">
    <span class="fas fa-github fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="https://twitter.com/joycepra_">
    <span class="fas fa-twitter fa-lg"></span>
     
  </a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="research.html">Research</a>
</li>
<li>
  <a href="pubs.html">Publications</a>
</li>
<li>
  <a href="Sci_life.html">Scientist Life</a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">




</div>


<p><img src="files/barco2.png" style="width:100%; align="center"></p>
<p><br></p>
<div id="research" class="section level1">
<h1>Research</h1>
<p><br> <img src="files/rato.JPG" style="width:40%; margin-right: 30px" align="left"></p>
<p><br></p>
<font size="5">
<div style="text-align: justify">
My work is related to understanding how abiotic and biotic factors interact and shape the patterns of genetic, morphological and ecological variation in small mammals
</div>
<p></font></p>
<p><br> <br><br></p>
<font size="6">
<div style="text-align: center">
POSTDOCTORAL RESEARCH
</div>
<p></font></p>
<font size="5">
<div style="text-align: justify">
The effect of climate change on the pattern of distribution, demography and genetic structure in neotropical semi-aquatic rodents.
</div>
<p></font></p>
<p><img src="files/holo2.png" style="width:35%; margin-left: 30px" align="right"></p>
<font size="3">
<div style="text-align: justify">
Similar ecological restrictions imposed by wetlands in South American biomes structure the genetic and demographic variation between taxa in a similar way. However, due to current and historical differences in these areas, species may reflect specific variation patterns for each biome they inhabit. Alternatively, taxa with shared natural histories may exhibit concordant patterns of variation, despite different biomes. To assess which of these hypotheses best explains the genetic structure and demographic history in three species of the genus <em>Holochilus</em>, we used New Generation Sequencing (ddRadSeq), along with analytical techniques that provide spatially explicit information about how genomic variation is structured across the landscape. Results are coming soon!!!
</div>
<p></font></p>
<p><br></p>
<font size="6">
<div style="text-align: center">
RESEARCH INTERESTS
</div>
<p></font></p>
<p><font size="5">Comparative Phylogeography</font></p>
<font size="4">
<div style="text-align: center">
<strong>Similar but different: Revealing the relative roles of species‐traits versus biome properties structuring genetic variation in South American marsh rats <a href="https://doi.org/10.1111/jbi.13529">Prado et al., 2019</a></strong>
</div>
<p></font></p>
<p><br></p>
<p><img src="files/fig_proc.png" style="width:55%; margin-right: 15px" align="left"></p>
<font size="3">
<div style="text-align: justify">
My PhD dissertation had already demonstrated how the genetic variation in Holochilus species was structured throughout geography, revealing that there is a significant association between genetic variation and geography for all species. Similarity in the strength of the association suggests connectivity patterns dictated by shared species‐traits predominate at the biome scale. However, substantial amounts of genetic variation are not explained by geography. Focusing on this portion of the variance, we demonstrate a significant quantitative association between genetic variation and the environmental space of a biome, and a qualitative association with varying regional stability. Specifically, historically stable areas are correlated with local levels of geographic structuring, suggesting that local biome‐specific histories affect population isolation/connectivity.
</div>
<p></font></p>
<p><br></p>
<p><font size="5">Species Delimitation</font></p>
<font size="4">
<div style="text-align: center">
<strong>New species boundaries and diversification history of marsh rat taxa clarify historical connections among ecologically and geographically distinct wetlands of South America <a href="https://doi.org/10.1016/j.ympev.2020.106992">Prado et al., 2021</a></strong>
</div>
<p></font></p>
<p><br></p>
<font size="3">
<div style="text-align: justify">
Taxa with broad geographic ranges that occur in different biomes and exhibit plastic morphological traits and/or adaptations to particular habitats make inferences about species boundaries especially challenging. However, technological and conceptual advances in the generation and analysis of genomic data have advanced the description of biodiversity. In this study (Prado et al., 2020) we address the outstanding questions about the delimitation of species in the genus <em>Holochilus</em>, a rodent with morphological specializations to wetland habitats, distributed through almost all the South America continent using genome‐wide SNP and morphometric data. Specifically, we apply a Bayesian model‐based species delimitation that revealed significant re-arrangements of species boundaries based on consideration of both morphometric and genomic data alone, or in combination. With these shifts in species boundaries, our results provide an insightful framework for considering putative dispersal corridors and debate surrounding their role in connecting disjunct South American biomes. Because of the ecological constraints of the marsh rats, and with the proposed taxonomic re-arrangements, the significance of our findings extends beyond systematics and suggests how diversification might be associated with past ecological/environmental linkages between disparate biomes during the Pleistocene. All the scripts and database of this manuscript is available on my <a href="https://github.com/joycepra/">github</a>.
</div>
<p></font></p>
<p><br></p>
<p><img src="files/Graphical_abstract2.pdf" style="width:70%" align="center"></p>
<p><br></p>
<p><font size="5">Biogeography</font></p>
<font size="4">
<div style="text-align: center">
<strong>Open vegetation dynamics and its role in structuring the diversity of sigmodontine rodents (Prado et al., in prep)</strong>
</div>
<p></font></p>
<p><br></p>
<p><img src="files/biogeo.png" style="width:65%; margin-right: 15px" align="left"></p>
<font size="3">
<div style="text-align: justify">
Identifying mechanisms that promote lineage diversification is essential to understand biogeographic patterns. In this study we investigate the biogeographic processes related to the extant genera of the Oryzomyini clade D. This clade includes the most comprehensive generic diversity of the tribe, with lineages occupying distinct phytophysiognomies. The current distributional pattern and the presence of tetralophodonts genera in this clade suggest that there is a close relationship between these genera and the evolution of South Amercia open areas. Using a genomic dataset we construct a time calibrated tree, and estimate the range evolution through statistical model testing among different biogeographic models. The origin of clade D occurred approximately 3.5 Ma, and most of diversification events occurred during Pliocene. Geographic range estimates suggest that the ancestors of this clade were open area inhabitants of the Cis-Andean part, and the retraction and expantion of the open enviroments, together with the formation of Isthmus of Panama, favored the dispersion and diversification on this clade.This study is still in prep!!!
</div>
<p></font></p>
<p><br></p>
<p><font size="5">Evolutionary Ecology</font></p>
<font size="4">
<div style="text-align: center">
<strong>Phenotypic and genomic differences between biomes of the South America marsh rat, <em>Holochilus brasiliensis</em> <a href="https://doi.org/10.1093/biolinnean/blab132">Prado et al., accepted</a></strong>
</div>
<p></font></p>
<p><br></p>
<p><img src="files/NICHE.png" style="width:65%; margin-right: 15px" align="left"></p>
<font size="3">
<div style="text-align: justify">
Abiotic factors can influence genetic and phenotypic divergence in several ways. Identifying which mechanisms are responsible for generating this variation is challenging given the necessity of high-resolution genomic data and experimental approaches. However, when evaluated in combination, microhabitats characteristics, genetic information and phenotypic variation can help us to understand how microhabitat preferences can influence the evolution of morphological and genetic patterns exhibited by taxa with broad distribution between very distinct biomes, such as the Atlantic Rainforest and Pampas biomes in South America. By combining distributional, environmental, phenotypic, and genomic information from a high habitat specialist semi-aquatic rodent (Holochilus brasiliensis), we quantified the relationship between microhabitat differences and the phenotypic and genetic variation. Results demonstrate differences in the fundamental niche between biomes, but we were not able to refute the hypothesis of niche conservatism or divergence. Additionally, despite differences in the connectivity patterns in both biomes, the genetic differences are not pronounced. In contrast, phenotypic data recovered significant differences related to overall size, and in all cases specimens from the Pampas were the largest.
</div>
<p></font></p>
<p><br></p>
<p><font size="5">Taxonomy</font></p>
<font size="4">
<div style="text-align: center">
<strong>A new species of South America marsh rat (<em>Holochilus</em>, Cricetidae) from northeastern Brazil <a href="https://doi.org/10.1093/jmammal/gyab104">Prado et al., accepted</a></strong>
</div>
<p></font></p>
<p><br></p>
<p><img src="files/Fig.9_oxe.pdf" style="width:50%; margin-right: 15px" align="right"></p>
<font size="3">
<div style="text-align: justify">
Marsh rats of the genus Holochilus are broadly distributed and inhabit several distinct environments throughout South America. As an enigmatic group with a somewhat uncertain taxonomy, the composition and names of taxa have shifted throughout history, including the boundaries of Holochilus sciureus, a species formerly distributed in the lowlands of the Guianas, Peru, Bolivia, and northern, central, and northeastern of Brazil. Based on a combination of morphological and morphometric traits, and genomic sequences, we redefined species boundaries and split this wide concept of <em>H. sciureus</em> into three separated species, including a redefined <em>H. sciureus</em>, the newly erected from synonymy <em>H. nanus</em>, and an unnamed taxon. We describe this unnamed species, and provide emended diagnoses for the two redefined species. The newly named taxon inhabits the northeastern part of Brazil and differs from the other congeners by a unique combination of phenotypic and genomic characters.
</div>
<p></font></p>
<p><br></p>
<font size="4">
<div style="text-align: center">
<strong>Systematic Studies of the Genus <em>Aegialomys</em> Weksler et al., 2006 (Rodentia: Cricetidae: Sigmodontinae): Geographic Variation, Species Delimitation, and Biogeography <a href="https://doi.org/10.1007/s10914-016-9360-y">Prado et al., 2018</a></strong>
</div>
<p></font></p>
<p><br></p>
<p><img src="files/Aegi.pdf" style="width:50%; margin-right: 15px" align="left"></p>
<font size="3">
<div style="text-align: justify">
<em>Aegialomys</em> occurs in open habitats west of the Ecuadorean and Peruvian Andes, including the Galapagos Archipelago. This genus currently includes two species, <em>A. galapagoensis</em> and <em>A. xanthaeolus</em>. We studied patterns of geographic variation to characterize the morphologic and morphometric variation and recognize diagnosable clusters of samples. Employing this evidence, within a phylogenetic framework employing morphological, molecular, and concatenated matrices, we diagnose monophyletic lineages and assign the appropriate names to species–group taxa. Qualitatively, we noted geographic variation in some characters, and quantitatively there is a pronounced increase in cranial dimensions along the north–south distribution axis, revealing the existence of four distinct clusters: North, South, Extreme South, and Galapagos. These results, along with the phylogenetic relationships, allowed us to hypothesize that <em>Aegialomys</em> exhibits four monophyletic species that we call: <em>Aegialomys galapagoensis</em>, restricted to the Galapagos Archipelago; <em>Aegialomys xanthaeolus</em>, distributed from Ecuador to northern Peru; <em>Aegialomys baroni</em>, ocurring in Central Perú; and <em>Aegialomys ica</em>, distributed in southern Peru. Our distributional data suggest that species discontinuities are associated with some well-known barriers in the western portion of South America. Through the Andes and trans–Andean area, there are some geographic features or areas, the Huancabamba Depression, that historically played a key role as barriers to plant and animal dispersion or as a boundaries to species distribution.
</div>
<p></font></p>
</div>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
